connectors/sources/mongo.py

```diff
@@ -145,6 +163,28 @@ def get_default_configuration(cls):
                 "type": "bool",
                 "value": False,
             },
+            "ssl_enabled": {
+                "display": "toggle",
+                "label": "SSL/TLS Connection",
+                "order": 7,
+                "type": "bool",
+                "value": False,
+            },
+            "ssl_ca": {
+                "depends_on": [{"field": "ssl_enabled", "value": True}],
+                "label": "Certificate Authority (.pem)",
+                "order": 8,
+                "required": False,
+                "type": "str",
+            },
+            "tlsinsecure": {
```

Comment:
Do you think we need this config?

From the [doc](https://pymongo.readthedocs.io/en/4.6.0/api/pymongo/mongo_client.html#pymongo.mongo_client.MongoClient):
>tlsInsecure: (boolean) Specify whether TLS constraints should be relaxed as much as possible. Setting `tlsInsecure=True` implies `tlsAllowInvalidCertificates=True` and `tlsAllowInvalidHostnames=True`. Defaults to `False`. **Think very carefully before setting this to `True` as it dramatically reduces the security of TLS.**

Reply:
For a user who does not have certificate data and is not aware of TLS, they can enable it. In Mongo Comapass, we have this option in the SSL/TLS connection. So, I added it to our connector. but, Yes, we can remove this if it lowers security. so, what do you suggest?

Reply:
What about making the config "advanced"? 
We've definitely found users who want to be able to ignore certificate validation, especially when just doing POCs on their laptops. But I agree with you, Chenhui, that we should give every signal that this setting should not be adjusted.

Reply:
For now, I have made this field `advanced`. @wangch079 Please share your reviews about this field.